docs/Tutorials/Kiln.md

```diff
@@ -0,0 +1,96 @@
+---
+Description: How to run Besu and Teku on Kiln
+---
+
+# Test Besu and Teku on Kiln
+
+You can test Besu as an [execution client](../Concepts/Merge.md) and [Teku](https://docs.teku.consensys.net/en/stable/)
+as a [consensus client](../Concepts/Merge.md) on the [Kiln testnet](https://blog.ethereum.org/2022/03/14/kiln-merge-testnet/).
+
+## Prerequisites
+
+1. Install [Besu](../HowTo/Get-Started/Installation-Options/Options.md) and
+   [Teku](https://docs.teku.consensys.net/en/stable/HowTo/Get-Started/Installation-Options/Install-Binaries/).
+
+1. Download the [configuration files](https://github.com/eth-clients/merge-testnets/tree/main/kiln):
```

Comment:
Why do we need these files? What do they contain?

Reply:
Seems these are not needed for Besu + Teku - removing.